fuzz/fuzz_targets/msg_targets/utils.rs

```diff
@@ -36,3 +51,84 @@ macro_rules! test_msg_hole {
 		}
 	}
 }
+
+#[macro_export]
+macro_rules! test_msg_writeable {
+	($MsgType: path, $data: ident) => {
+		{
+			use lightning::ln::msgs::{MsgEncodable, MsgDecodable};
+			use lightning::util::ser::{Writer, Reader, Writeable, Readable};
+			let mut r = Reader::new(::std::io::Cursor::new($data));
+			if let Ok(msg) = <$MsgType as Readable<::std::io::Cursor<&[u8]>>>::read(&mut r) {
+				let p = r.get_ref().position() as usize;
+				let mut w = Writer::new(::std::io::Cursor::new(vec![]));
+				msg.write(&mut w).unwrap();
+
+				let buf = w.into_inner().into_inner();
+				assert_eq!(&r.into_inner().into_inner()[..p], &buf[..p]);
+
+				let encoded = <$MsgType as MsgDecodable>::decode(&buf[..]).unwrap().encode();
+				assert_eq!(&$data[..p], &encoded[..]);
+			}
+		}
+	}
+}
+
+#[macro_export]
+macro_rules! test_msg_writeable_simple {
```

Comment:
This is unused, no?

Reply:
This is used for Ping/Pong message. Wanted to remove ping/pong message test in the top fuzz directory.